.github/workflows/pytest.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
```

Comment:
run on push is maybe a bit to much. Especially on any branch.
As a nice example, this build now runs twice for this particular pull request :)

Reply:
Changed to on PR